antenna-sw360-adapter/src/test/java/org/eclipse/sw360/antenna/sw360/rest/SW360ProjectClientTest.java

```diff
@@ -42,6 +42,10 @@
 
 public class SW360ProjectClientTest {
     private static final String REST_URL = "http://localhost:8080/resource/api";
+    private static final boolean PROXY_ENABLE = false;
+    private static final String PROXY_HOST = "localhost";
```

Comment:
this is now unnecessary?

Reply:
here we probably need those values, since we call `SW360ProjectClient` constructor directly and I guess that we do not have `context` available here, or do we?